src/app/blog/components/post-detail/post-detail.component.ts

```diff
@@ -74,13 +74,13 @@ export class PostDetailComponent implements OnInit {
   // the theDta is getting the commentForm values
   postYourComment(theData) {
     const message = theData.msg;
-    const author = theData.name;
+    const author = theData.author;
 
     const createdDate = this.now; // here i am assigning the values of the new Date to createdDate -->
```

Comment:
**Comment here is better :**  assigning the values of the new Date.
here i am :  Nobody wants to know you 
to createdDate: This redundance / Duplicata , the variable cal createdDate

Reply:
ok i will remove it